pytorch_lightning/core/decorators.py

```diff
@@ -11,12 +11,18 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-"""Decorator for LightningModule methods."""
+from pytorch_lightning.utilities import rank_zero_deprecation
 
-from functools import wraps
-from typing import Callable
+rank_zero_deprecation(
```

Comment:
are you sure you want to add this here? shouldn't it be in the function?
one should be allowed to import other functions from this module.

Reply:
Yup, there's only `parameter_validation` in the module now. We could always move it inside the function if required.